### ISSUE TITLE
Writing metmask script as module

### ISSUE THREAD

**Issue by kozo2** [hredestig/metmask#1]
To support Windows,
I'd like to ask you to write `scripts/metmask` as Python module.

(I think that using py2exe is not good now.)

**Comment by hredestig** [comment]
Hey, thanks for the interest! I just had a reason to revisit this old project and plan to bring it to modern times. It is actually a module already and at least on Mac / Linux I could `pip install <dir>` it.  Not sure about Windows. Anyway, stay tuned..

**Comment by kozo2** [comment]
It seems that metmask does not work as module on Windows (conda) env.
```
C:\Users\knishida\Documents\GitHub>C:\Users\knishida\Miniconda2\Scripts\pip.exe install metmask -U
Requirement already up-to-date: metmask in c:\users\knishida\miniconda2

C:\Users\knishida\Documents\GitHub>C:\Users\knishida\Miniconda2\Scripts\pip.exe list
DEPRECATION: The default format will switch to columns in the future. You can use --format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.
cffi (1.9.1)
conda (4.3.11)
cryptography (1.7.1)
defusedxml (0.5.0)
enum34 (1.1.6)
idna (2.2)
ipaddress (1.0.18)
menuinst (1.4.4)
metmask (0.5.3)
pip (9.0.1)
pyasn1 (0.1.9)
pycosat (0.6.1)
pycparser (2.17)
pyOpenSSL (16.2.0)
pywin32 (220)
requests (2.12.4)
setuptools (27.2.0)
six (1.10.0)
SOAPpy (0.12.22)
wheel (0.29.0)
wstools (0.4.5)

C:\Users\knishida\Documents\GitHub>C:\Users\knishida\Miniconda2\python.exe -m pip --version
pip 9.0.1 from C:\Users\knishida\Miniconda2\lib\site-packages (python 2.7)

C:\Users\knishida\Documents\GitHub>C:\Users\knishida\Miniconda2\python.exe -m metmask --version
C:\Users\knishida\Miniconda2\python.exe: No module named metmask.__main__; 'metmask' is a package and cannot be directly executed
```

**Comment by kozo2** [comment]
It seems that metmask can not be used as a module not only in Windows but also in Linux environment.

```
root@DESKTOP-ETL8RDS:~/gh# pip2 install ./metmask -U
Processing ./metmask
Installing collected packages: metmask
  Found existing installation: metmask 0.5.3
    Uninstalling metmask-0.5.3:
      Successfully uninstalled metmask-0.5.3
  Running setup.py install for metmask ... done
Successfully installed metmask-0.5.3
root@DESKTOP-ETL8RDS:~/gh# python2 -m metmask --version
/usr/bin/python2: No module named metmask.__main__; 'metmask' is a package and cannot be directly executed
root@DESKTOP-ETL8RDS:~/gh# cd metmask
root@DESKTOP-ETL8RDS:~/gh/metmask# git log HEAD
commit d82071e98e22d4ea94897874f973af61f6fd3641
Author: Henning Redestig <henning.red@gmail.com>
Date:   Mon Mar 20 22:14:30 2017 +0100

    isort
```

**Comment by hredestig** [comment]
Ah! Sorry I didn't get what you were after first, indeed you're right, the CLI script can't be used as module. I'll fix.

**Comment by kozo2** [comment]
Thanks.
Now it's common to install multiple Python environments on one computer,
for such environments this metmask call way is useful.

**Comment by hredestig** [comment]
Okay this should be fixed with ee0be14f5a4970a10f3cc9ad98dd847c1b9a251b 

Please confirm and close

**Comment by kozo2** [comment]
Thank you very much!
I checked that the master HEAD worked well in my Windows+Miniconda2 environment.

```
~\Documents\GitHub\metmask [master ≡ +2 ~0 -0 !]> g pull
Already up-to-date.
~\Documents\GitHub\metmask [master ≡ +2 ~0 -0 !]> C:\Users\knishida\Miniconda2\Scripts\pip.exe install .
Processing c:\users\knishida\documents\github\metmask
Installing collected packages: metmask
  Found existing installation: metmask 0.5.3
    Can't uninstall 'metmask'. No files were found to uninstall.
  Running setup.py install for metmask ... done
Successfully installed metmask-0.5.4
~\Documents\GitHub\metmask [master ≡ +2 ~0 -0 !]> cd ~/
~> .\Miniconda2\python.exe -m metmask --version
This is metmask v0.5.4
~> .\Miniconda2\python.exe -m metmask --help
Usage: __main__.py [X] [options]
    X represents a stream of identifiers from STDIN
    or a single identifier if -a is set or a file with
    identifiers, one per line. If X is not provided one
    or more of the options below must be set.

Options:
```
